tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -2681,35 +2683,50 @@ private String handleCurrencyFormat(XPathParts parts, String value, boolean show
+            df.applyPattern(value);
             return formatCountDecimal(df, countValue);
         }
 
-        double sampleAmount = 1295.00;
-        example = addExampleResult(formatNumber(df, sampleAmount), example, showContexts);
-        example = addExampleResult(formatNumber(df, -sampleAmount), example, showContexts);
-
-        if (showContexts && !altAlpha) {
-            // If this example is not for alt="alphaNextToNumber", then if the currency symbol
-            // above has letters (strong dir) add another example with non-letter symbol
-            // (weak or neutral), or vice versa
-            if (symbolIsLetters(currencySymbol, false)) {
-                currency = "EUR";
-                checkPath = "//ldml/numbers/currencies/currency[@type=\"" + currency + "\"]/symbol";
-                currencySymbol = cldrFile.getWinningValue(checkPath);
-            } else {
-                currencySymbol = currency;
-            }
-            df = icuServiceBuilder.getCurrencyFormat(currency, currencySymbol, numberSystem);
-            df.applyPattern(value);
-            example = addExampleResult(formatNumber(df, sampleAmount), example, showContexts);
-            example = addExampleResult(formatNumber(df, -sampleAmount), example, showContexts);
+        String symbol_eur;
+        String symbol_usd;
+        boolean symbol_first = value.charAt(0) == '¤';
+        if (altAlpha) {
+            symbol_eur = "EUR";
+            symbol_usd = symbol_first ? "$USD" : "USD$";
+        }
+        else {
+            symbol_eur = "€";
+            symbol_usd = symbol_first ? "USD$" : "$USD";
         }
+        
+        String pattern_eur = value.replace("¤", symbol_eur);
+        String pattern_usd = value.replace("¤", symbol_usd);
+       
+        DecimalFormat df_eur = icuServiceBuilder.getNumberFormat(pattern_eur, numberSystem);
+        DecimalFormat df_usd = icuServiceBuilder.getNumberFormat(pattern_usd, numberSystem);
+
+        double sampleAmount = 1295.00;
+        example = addExampleResult(formatNumber(df_eur, sampleAmount), example, showContexts);
+        example = addExampleResult(formatNumber(df_eur, -sampleAmount), example, showContexts);
+        example = addExampleResult(formatNumber(df_usd, sampleAmount), example, showContexts);
+        example = addExampleResult(formatNumber(df_usd, -sampleAmount), example, showContexts);
+
+        DecimalFormat df = icuServiceBuilder.getCurrencyFormat(currency, "¤", numberSystem);
+        df.applyPattern(value);
+        example =
+                addExampleResult(
+                        formatNumber(df, sampleAmount).replace("¤", currencySymbol),
+                        example,
+                        showContexts);
+        example =
+                addExampleResult(
+                        formatNumber(df, -sampleAmount).replace("¤", currencySymbol),
+                        example,
+                        showContexts);
```

Comment:
Where do things stand with this draft PR? it adds more examples, using df_eur and df_usd...

@srl295 I see you made a more recent commit (91e10790f6c) to this method: it returned String but is now void and ends with `examples.add(example)`

Emiyare also made a more recent commit (00e1fc23f1d) inserting a few lines into this method. It looks like the old changes could be wrapped into the current code fairly easy but not just by rebasing...